lib/daru/view/adapters/googlecharts/base_chart.rb

```diff
@@ -27,13 +27,10 @@ def draw_js_chart_wrapper(data, element_id)
       js << "\n  function #{chart_function_name(element_id)}() {"
       js << "\n  \t#{@data_table.to_js}"
       js << "\n  \tvar wrapper = new google.visualization.ChartWrapper({"
-      js << "\n  \t\tchartType: '#{chart_name}',"
-      js << append_data(data)
-      js << "\n  \t\toptions: #{js_parameters(@options)},"
-      js << "\n  \t\tcontainerId: '#{element_id}',"
-      js << "\n  \t\tview: #{extract_option_view}"
+      js << extract_chart_wrapper_options(data, element_id)
```

Comment:
ChartWrapper here ?

Reply:
`add_listeners_js('wrapper')` in this method was generating rubocop error so I've created another method `extract_chart_wrapper_options` in the common module display.